.coveragerc

```diff
@@ -85,6 +85,9 @@ omit =
     homeassistant/components/hive.py
     homeassistant/components/*/hive.py
 
+    homeassistant/components/hue.py
```

Comment:
You are adding tests (thank you!) so no need to add the files to .coveragerc

Reply:
So should I drop all the changes to .coveragerc or only this line (`homeassistant/components/hue.py`)?
`homeassistant/components/light/hue.py` was already in the file some way down, should I live it there?

Reply:
.coveragerc includes file not covered by tests, so remove both homeassistant/components/hue.py and homeassistant/components/*/hue.py